backend/src/main/java/ca/bc/gov/hlth/mohums/util/FilterUserByOrgId.java

```diff
@@ -81,7 +81,7 @@ static String extractOrgId(Object orgDetails) {
             // These exceptions are thrown when the value found by EXPRESSION
             // is not a valid JSON object. We can't stop people from creating
             // 'org_details' attributes that don't contain JSON data.
-            LOGGER.warn("Unable to parse: \"{}\": {}.", organizationDetails, e);
```

Comment:
This would always log a full stacktrace for a couple tests when building which makes you do a double take for no reason if you're reviewing logs.

Reply:
I can't recall what I found, but it would be nice if we could do this with a check instead of an exception.